### PR TITLE
Update .css to make the design consistent

### PR DESCRIPTION
**Pull request by jonezkti**
Updated the design of itembox on item use and when showing required items to use the same design as the slots in the inventory.

Required before:
req-old
Required after:
req-new

Using item before: (Same box appears when an item is removed)
use-old
Using item after:
use-new

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes
- Does your code fit the style guidelines? Yes
- Does your PR fit the contribution guidelines? Yes

### CODE REVIEW

File: html/css/main.css [hunk 1]
```diff
@@ -461,72 +463,89 @@ body {
...
+	color: white;
+	font-size: 0.8vh;
+	padding: 3px;
+	text-align: center;
+	letter-spacing: 1px;
+	font-family: 'Poppins', sans-serif;
 }
 
 .requiredItem-container {
-	display: none;
```
Comment (xchopin) [inline html/css/main.css:486]: No, here you are breaking the quantity field man

**Comment by xchopin** [comment]
I have tried your PR it breaks the inventory by not being able to choose quantities. I fixed it by adding back the display: none (cf my comment)

**Comment by jonezkti** [comment]
Good catch, thanks for noticing the mistake. Fixed it, everything should now be working correctly.

**Review by dexter9006 — APPROVED** [review]

**Review by JackWemble — APPROVED** [review]

**Comment by jeffreykolb** [comment]
Didn't work, label is doing weird on use
afbeelding

**Pull request closed without merging**